enclave-runtime/Cargo.lock

```diff
@@ -686,20 +679,21 @@ dependencies = [
 
 [[package]]
 name = "frame-metadata"
```

Comment:
We now have two frame-metadata sources. As parity seems to be using the one from crates.io, we should probably switch as well.

i.e. switching https://github.com/integritee-network/worker/blob/master/core-primitives/storage/Cargo.toml#L19
to https://github.com/paritytech/substrate/blob/master/frame/support/Cargo.toml#L19

Reply:
Yes, the api-client switched too.